src/patternfly/components/Check/check-label.hbs

```diff
@@ -3,4 +3,7 @@
     {{{check-label--attribute}}}
   {{/if}}>
   {{> @partial-block}}
+{{#if check--IsRequired}} 
+  {{> check-label-required}}
+{{/if}}
 </{{#if check-label--type}}{{check-label--type}}{{else}}label{{/if}}>
```

Comment:
Just one nit! Need to indent this block since it's nested

```suggestion
  {{> @partial-block}}
  {{#if check--IsRequired}} 
    {{> check-label-required}}
  {{/if}}
</{{#if check-label--type}}{{check-label--type}}{{else}}label{{/if}}>
```